### PR TITLE
Minor Fixes and Updates

### PR DESCRIPTION
**Pull request by Jacques00**
### Bug/Typo Fixes
* Various corrections and typo fixes.
* Spelling "mavellous".

### Updates/Edits
* Add three new item flags for chest, groin, and ass exposure for semi-covered items (armor/clothing/underwear).
* Add a "heldWeapon" parser.
* Bad Ends are now more explicit, with the changes showing up in the appearance, where applicable. Also includes time and status effect changes.

### New Content
* Thare Plantation with three possible sex scenes with a twink zil, by Nonesuch.
* Functions to remove items (into the void). Can remove from slots individually with specification strings or wholesale. Includes: removeClothes(), removeEquipment(), removeAll().

### CODE REVIEW

File: includes/mhenga/tharePlantation.as [hunk 1]
```diff
@@ -0,0 +1,1031 @@
...
+			output("\n\n<i>“And so far,”</i> he goes on in a harder tone, <i>“only Mr. Darnock’s company has reached out to us. Not the people of the place you call Esbeth, who imprison us or drive us away with their giant white stings; not those who call themselves the Zen jinn, who covet the sirens’ dark secrets and kill any that stand in their way; and not the terrible black wings, who stalk this land now, spiriting our folk away.”</i>");
+			output("\n\n<i>“So why haven’t more zil taken him up on his offer?”</i> you say, looking over the tub’s rim at him.");
+			output("\n\n<i>“Um. I’m not </i>typical<i> of zil. I don’t think Mr. Darnock understands this,”</i> the slim zil says, shuffling his carapace-covered feet. <i>“For most zil, in order to get them to obey or even listen to you, first you have to fight them, or make them so aroused they’ll do whatever you say. I never liked that, because I preferred submitting straight away. It just feels so much better, you know? So I was happy to take orders from sir. My brothers and sisters though... for them it was a non-starter, because he refused to fight or have sex with them first.”</i>");
```
Comment (Fenoxo) [inline includes/mhenga/tharePlantation.as]: The whole "forcey funtimes" is an outside of the village thing. Inside their villages they are not nearly so savage with each other. Goddamnit, Nonesuch.

Comment (Fenoxo) [inline includes/mhenga/tharePlantation.as]: Do you want to adjust this, or shall I?

**Comment by Fenoxo** [comment]
Goddamn, you are a rock star.

File: includes/mhenga/tharePlantation.as (hunk 1 above)
Comment (Jacques00) [inline includes/mhenga/tharePlantation.as]: Feel free to edit--If you really want to make sure, i guess you can run it buy Nonesuch.

Comment (Fenoxo) [inline includes/mhenga/tharePlantation.as]: Don't need to. I'll wing it.

**Pull request merged**